### PR TITLE
Create 2016-11-14-apps-on-ipfs.md

### PR DESCRIPTION
**Pull request by VictorBjelkholm**

### CODE REVIEW

File: meeting-notes/2016-11-14-apps-on-ipfs.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+* new people contributing! Thanks to all of them
+* Looking at the roadmap, might move the product design milestone to after the web version, to push js-ipfs forward
+
+* Hard to demo Orbit today, because of a small number of bugs
+* Work is being done on getting a more configurable Orbit, to allow multiple instances on the same computer
+* Troubles with connectivity, peers not connecting always with pubsub/floodsub
+
+### Write-controlled feeds
+
+* Prototype for data structures for access control!
```
Comment (haadcode) [inline meeting-notes/2016-11-14-apps-on-ipfs.md]: Access controlled data structures :)

**Comment by haadcode** [comment]
One small change request, otherwise LGTM

**Comment by VictorBjelkholm** [comment]
@haadcode thanks! Fixed :)

**Comment by haadcode** [comment]
Thank you @VictorBjelkholm!

**Pull request merged**